docs/demos/create-br1-linux-bridge.yaml

```diff
@@ -0,0 +1,20 @@
+spec:
```

Comment:
this is confusing, the yaml should be full and usable on cluster. looking at this I don't know where to place this

Reply:
As we have talk about the created/modified NodeNetworkState is in the same namespace as the desiredState we change so we have to edit exactly this or do a patch.